lib/main_common.pm

```diff
@@ -369,7 +369,7 @@ sub load_boot_tests {
 
 sub load_reboot_tests {
     # there is encryption passphrase prompt which is handled in installation/boot_encrypt
-    if (check_var("ARCH", "s390x") && !(get_var('ENCRYPT') && check_var('BACKEND', 'svirt'))) {
+    if (check_var("ARCH", "s390x") && !get_var('ENCRYPT')) {
```

Comment:
careful, svirt is also used e.g. by xen tests

Reply:
Here we are safe, as have condition for the architecture.